### PR TITLE
Nightmare Clock Tower Dungeon

### PR DESCRIPTION
**Pull request by Ridley8819**
- added mobs and (updated) skills
- added quests_aldebaran.txt
- iro dialogues
- all items are already in available in item_db.conf
- maps are already available in mapcache, index and .conf

Sources:
https://www.youtube.com/watch?v=W-cDlkq5aoE
http://ro.vrc7.net/?page_id=1326
http://www.divine-pride.net/database/map/c_tower2_
http://www.divine-pride.net/database/map/c_tower3_

---
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1389)

### CODE REVIEW

File: db/re/mob_skill_db.txt [hunk 1]
```diff
@@ -10024,68 +10024,73 @@
...
+2921,Owl Viscount@NPC_EMOTION,idle,197,1,2000,0,5000,yes,self,always,0,6,0x37B5,,,,,
+2923,Owl Marquees@AL_TELEPORT,idle,26,1,10000,0,0,yes,self,rudeattacked,,,,,,,,
+2923,Owl Marquees@MG_LIGHTNINGBOLT,attack,20,3,2000,0,5000,yes,target,always,0,,,,,,,
+2923,Owl Marquees@MG_LIGHTNINGBOLT,chase,20,3,2000,0,5000,yes,target,always,0,,,,,,,
+2923,Owl Marquees@NPC_STONESKIN,attack,675,3,10000,0,30000,yes,self,always,,,,,,,6,
+2923,Owl Marquees@NPC_EMOTION,idle,197,1,2000,0,5000,yes,self,always,0,32,,,,,,
+2923,Owl Marquees@NPC_EMOTION,chase,197,1,200,0,5000,yes,self,always,0,19,0xA1,,,,,
+2923,Owl Marquees@NPC_EMOTION,idle,197,1,2000,0,5000,yes,self,always,0,6,0x37B5,,,,,
+2923,Owl Marquees@NPC_SUMMONSLAVE,idle,196,2,10000,2000,60000,no,self,slavele,0,2921,,,,,,
+3074,Time Holder@AL_TELEPORT,walk,26,1,5000,0,5000,yes,self,rudeattacked,,,,,,,,
```
Comment (Asheraf) [inline db/re/mob_skill_db.txt]: as i know Time Holder doesn't have the teleport skill

Comment (Ridley8819) [inline db/re/mob_skill_db.txt]: you're right, i mixed something up here Oo

File: db/re/mob_db.conf [hunk 2]
```diff
@@ -81931,15 +81931,353 @@ mob_db: (
...
+	AttackMotion: 824
+	DamageMotion: 440
+	Drops: {
+		Tatters_Clothes: 2000
+		Soft_Feather: 1500
+		Kakkung_: 1
+		Staff_Of_Soul: 1
+		Menswear: 3
+		Sword_Stick: 1
+		Monocle_: 1
```
Comment (Asheraf) [inline db/re/mob_db.conf]: according to kRO website it drops the item 18874 which in the item db Cyclops_Glasses !!

File: npc/re/quests/quests_aldebaran.txt [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+//=      | |_| | ___ _ __ ___ _   _| | ___  ___
+//=      |  _  |/ _ \ '__/ __| | | | |/ _ \/ __|
+//=      | | | |  __/ | | (__| |_| | |  __/\__ \
+//=      \_| |_/\___|_|  \___|\__,_|_|\___||___/
+//================= License ===============================================
+//= This file is part of Hercules.
+//= http://herc.ws - http://github.com/HerculesWS/Hercules
+//=
+//= Copyright (C) 2012-2016  Hercules Dev Team
+//= Ridley
```
Comment (MishimaHaruna) [inline npc/re/quests/quests_aldebaran.txt]:
```
//= Copyright (C) 2016  Hercules Dev Team
//= Copyright (C) 2016  Ridley
```

**Comment by Ridley8819** [comment]
Review status: 0 of 6 files reviewed at latest revision, 3 unresolved discussions.

---

*[db/re/mob_db.conf, line 82277 \[r3\]](https://reviewable.io:443/reviews/herculesws/hercules/1389#-KOMmUKvjFWQwaip4k2p-r3-82277:-KPBk9GKX-hymF_Avn9y:b-896fix) ([raw file](https://github.com/herculesws/hercules/blob/e86aea5ce3d239d3ff919ea460946a5f849da2f3/db/re/mob_db.conf#L82277)):*
<details><summary><i>Previously, Asheraf (Acheraf Rian) wrote…</i></summary>
> according to kRO website it drops the item 18874 which in the item db Cyclops_Glasses !!
</details>
Done.

---

*[npc/re/quests/quests_aldebaran.txt, line 13 \[r1\]](https://reviewable.io:443/reviews/herculesws/hercules/1389#-KOMmUKvjFWQwaip4k2t-r1-13:-KPBk7BsAf5RODpONDVJ:b-896fix) ([raw file](https://github.com/herculesws/hercules/blob/283e66f0c74b404671a50b06710945b8ce04159b/npc/re/quests/quests_aldebaran.txt#L13)):*
<details><summary><i>Previously, MishimaHaruna (Haru) wrote…</i></summary>
> ```
> 
> //= Copyright (C) 2016  Hercules Dev Team
> 
> //= Copyright (C) 2016  Ridley
> 
> ```
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/herculesws/hercules/1389)*

File: npc/re/quests/quests_aldebaran.txt [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+		mes("Come back when you've reached ^0000FFLevel 150^000000.");
+		close;
+	}
+	mes("Time, memories, environments... all can become warped here.");
+	next;
+	mes("[Belljamin Button]");
+	mes("I'm  begging you not to ask more about it. It might put your life in danger.");
+	next;
+	switch (select("Don't ask", "Go to 2nd floor of warped tower", "Go to 3rd floor of warped tower")) {
+	case 1:
```
Comment (hemagx) [inline npc/re/quests/quests_aldebaran.txt]: i believe you should add tab before case as well.

Comment (dastgir) [inline npc/re/quests/quests_aldebaran.txt]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style
Chapter 1, the one here Ridley used is correct

Comment (Ridley8819) [inline npc/re/quests/quests_aldebaran.txt]: Yes, I replied that to him too, but In a pm :x

**Pull request closed without merging**